### DISCUSSION TITLE
help my fan control isnt showing anything

### DISCUSSION THREAD

**Discussion by hashimm2345** [Rem0o/FanControl.Releases#2483] [Q&A]
![Screenshot 2024-05-02 140446](https://github.com/Rem0o/FanControl.Releases/assets/168733397/0abc610f-c623-44c2-94f4-4af1a7768a2e)

**Comment by Rem0o** [comment]
Your computer may (most probably) not be compatible.

**Reply by hashimm2345** [reply]
can i do anything about it?

**Reply by Rem0o** [reply]
Telling us your specs.
